src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -22,11 +22,13 @@ import {
 interface Props {
   className?: string
   useSimplifiedBucketForm?: boolean
+  showCreateButton?: boolean
 }
 
 const WriteDataHelperBuckets: FC<Props> = ({
   className = 'write-data--details-widget-title',
   useSimplifiedBucketForm = false,
+  showCreateButton,
```

Comment:
this should default to `true` to keep the behavior of existing components using this that don't pass in `showCreateButton`.